mix.exs

```diff
@@ -29,7 +29,7 @@ defmodule Elsa.MixProject do
 
   defp deps do
     [
-      {:brod, "~> 4.4"},
+      {:brod, git: "https://github.com/simplifi/brod", ref: "7304dc33b524fe32aa620890ef773ebc3b0928c3"},
```

Comment:
```suggestion
      # Pointing to a fork until https://github.com/kafka4beam/brod/pull/625 gets merged in 
      {:brod, git: "https://github.com/simplifi/brod", ref: "7304dc33b524fe32aa620890ef773ebc3b0928c3"},
```

Reply:
It looks like that PR is just waiting on us to fix two failing jobs?

Reply:
@LiruMouse correct... although when Cristina initially submitted it, it was passing.  So I think some bit rot occurred there.  It will all get fixed up sooner or later I'm sure, but considering we aren't paying the brod maintainers for their work, they're gonna get around to it when they get around to it :)